cli/cli_app.go

```diff
@@ -549,9 +560,6 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) (finalStatus er
 		}
 	}()
 
-	// Run an init in case there are new modules or plugins to import
-	shell.NewTFCmd(terragruntOptions).Args([]string{"init", "--backend=false"}...).WithRetries(3).Output()
```

Comment:
We only need one init

Reply:
Why is moved before? And moreover, why it was call after the PreHooks that is called AfterInitState?

Reply:
It was called here because the other `init` wasn't called all the time. The other init would only be called to initiate the remote state and this one would be called all the time to fetch missing modules and plugins.

One init that we call on every run is enough, and as you say, it shouldn't be called after the hooks that are supposed to be called after init